spec/dummy/app/assets/config/manifest.js

```diff
@@ -1,5 +1,6 @@
-
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
-//= link govuk_publishing_components_manifest.js
+//= link component_guide/application.js
+//= link component_guide/application.css
```

Comment:
We should be able to have this as part of the govuk_publishing_components manifest